CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Precise UI Changelog
 
+## 2.1.0
```

Comment:
Either merge with 2.0.1 to 2.1.0 or append it to 2.0.1 (I think it should be merged).

Note that 2.0.1 was not released yet.